src/js/pages/settings/UISettingsPage.js

```diff
@@ -83,8 +101,60 @@ class UISettingsPage extends React.Component {
                 }
               />
             </ConfigurationMapSection>
+            <ConfigurationMapSection>
+              <ConfigurationRow
+                keyValue="refreshRate"
+                title={<Trans>Refresh Rate</Trans>}
+                value={`${Config.getRefreshRate() / 1000} ${i18nMark(
+                  t`seconds`
```

Comment:
Maybe in some languages, we don't say seconds after the number.

Reply:
that's ok

Reply:
Is there an example somewhere for the units being before or being after the value?

Reply:
@Poltergeist , in English there are after the value, in [Shona](https://en.wikipedia.org/wiki/Shona_language) (one example I found, there could be other), they are before - https://translate.google.com/?hl=en#view=home&op=translate&sl=en&tl=sn&text=five%20seconds%0Aten%20seconds .

Reply:
that's not important. Let's keep it like it is

Reply:
I fixed the issue. anyways. I was looking for a code example to make this possible. I found one.